game/core.py

```diff
@@ -311,6 +314,10 @@ def xmlrpc_query_state(self, request, game_id, player_token):
             result["whose_turn"] = game.turn.alias
             result["hand"] = player.hand
             result["top_card"], result["top_card_v"] = game.deck.top_card()
+
+            if(game.turn.token[0].islower()):
```

Comment:
more pythonic syntax please. `if game.turn.token[0].islower():`

Reply:
Did that, will be more "pythonic" from next time